data_collection/gazette/spiders/base/municipioonline.py

```diff
@@ -0,0 +1,49 @@
+import re
+from datetime import datetime
+
+import scrapy
+
+from gazette.items import Gazette
+from gazette.spiders.base import BaseGazetteSpider
+
+
+class BaseMunicipioOnlineSpider(BaseGazetteSpider):
+    allowed_domains = ["municipioonline.com.br"]
+
+    def start_requests(self):
+        url = f"https://www.municipioonline.com.br/{self.url_uf}/prefeitura/{self.url_city}/cidadao/diariooficial"
+        yield scrapy.Request(url, callback=self.post_request)
+
+    def post_request(self, response):
+        start_date = self.start_date.strftime("%d/%m/%Y")
+        end_date = self.end_date.strftime("%d/%m/%Y")
+
+        formdata = {
+            "__EVENTTARGET": "ctl00$body$btnBuscaPalavrachave",
+            "ctl00$body$txtDtPeriodo": f"{start_date}-{end_date}",
+        }
+
+        return scrapy.FormRequest.from_response(response, formdata=formdata)
+
+    def parse(self, response):
+        editions_list = response.xpath("//div[@class='panel panel-default']")
+
+        for edition in editions_list:
+            metadata = edition.xpath(".//div[@class='panel-title']//text()").get()
+            edition_number = re.search(r"(\d+)/", metadata).group(1)
+            raw_date = re.search(r"(\d{2}/\d{2}/\d{4})", metadata).group(1)
+            edition_date = datetime.strptime(raw_date, "%d/%m/%Y").date()
+
+            raw_path = edition.xpath(".//a[@onclick]").get()
+            url_path = re.search(r"l=(.+)'", raw_path).group(1)
```

Comment:
Mesmo comentário anterior
```suggestion
            url_path = edition.xpath(".//a[@onclick]").re_first(r"l=(.+)'")

```